src/presentation/ui/component/form/textArea.templ

```diff
@@ -29,3 +29,52 @@ templ TextArea(
 		}
 	</div>
 }
+
+script TextAreaReadOnlyLocalState() {
+	document.addEventListener('alpine:init', () => {
+		Alpine.data('textAreaReadOnly', () => ({
+			copyToClipboard(elementId) {
+				const textInputElement = document.getElementById(elementId);
```

Comment:
You couldn't use x-ref here as well? You said you would change that.

Reply:
I did even better. By adding `x-ref`, I was able to remove those lines and keep only the one necessary to copy the content to the clipboard through `x-ref`. This allowed me to remove the local state script and use it directly in `@click`. Zen programmer.